From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@nigele-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nigele). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

davidmu1 committed:
new article

davidmu1 committed:
acrolinx corrections

davidmu1 committed:
corrections

davidmu1 committed:
corrections

davidmu1 committed:
corrections

rachelappel committed:
remove app-service-linux-how-to-create-web-app, update toc

rachelappel committed:
remove images

mmacy committed:
Test commit

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

rojasja committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

rojasja committed:
Adding new IDP Scenarions to custom policies

rojasja committed:
Edit corrections

rojasja committed:
Error corrections 2nd try

rojasja committed:
Minor correction file reference

rojasja committed:
Updating TOC

rojasja committed:
Added Include WARNING to the new articles

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mmacy committed:
Merge branch 'master' of https://github.com/mmacy/azure-docs-pr

mmacy committed:
Bring up to date w/master

mmacy committed:
Bring up to date w/master. Again.

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

carolinacmoravia committed:
M8230: Possible incorrect terminology: **Device settings > Hardware health**

@alkohli,
It seems that it should say **Device settings > Hardware health** according to the article storsimple/storsimple-8000-device-modes.md. 
Could you, please, review this and reply?
Many thanks in advance.

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

carolinacmoravia committed:
M8269: Incorrect image structure

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/mmacy/azure-docs-pr

carolinacmoravia committed:
M8339: Repeated word: "policy"

carolinacmoravia committed:
M8342: Possible incorrect string: "Configure App Service web apps with Application Gateway"

@georgewallace. 
It seems it should read "Configure App Service web apps with Application Gateway." Could you confim this, please?
Many thanks in advance.

carolinacmoravia committed:
M8355: Incorrect bold type structure

carolinacmoravia committed:
M8356: Incorrect tag that block text content

itechedit committed:
edit pass: app-service-mobile-xamarin-forms-get-started

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into app-service-mobile-xamarin-forms-get-started

itechedit committed:
edit pass: app-service-mobile-xamarin-forms-get-started

ktoliver committed:
edit pass: active-directory-saas-helpscout-tutorial.md

ktoliver committed:
more edits

rojasja committed:
With changes suggested by human review

rojasja committed:
correcting one file get started custom

rojasja committed:
Merge branch 'master' into master

rojasja committed:
deleting unnecessary file

ktoliver committed:
edit pass: active-directory-saas-sapboc-tutorial.md

ktoliver committed:
more edits

ktoliver committed:
more edits

rojasja committed:
Merge branch 'master' into master

rojasja committed:
correct spelling Trust Framework

rojasja committed:
Correct ADFS file

rojasja committed:
corrections adfs file

rojasja committed:
Update active-directory-b2c-custom-setup-adfs2016-idp.md

rojasja committed:
Merge pull request #1 from rojasja/rojasja-patch-1

Update active-directory-b2c-custom-setup-adfs2016-idp.md

rojasja committed:
fixing display problems adfs file

rojasja committed:
correct goog file content

rojasja committed:
update adfs file indented picture references

rojasja committed:
corrected space after xml code

Line163 deleted space should now appear continuous number in 1...4,5

rojasja committed:
msa corrections on numbering during staging

rojasja committed:
Update active-directory-b2c-create-custom-attributes-profile-edit-custom.md

rojasja committed:
corrected ordering on adfs file

rojasja committed:
corrected numbering on goog idp instructions

rojasja committed:
corrected numbering

When numbering list you cannot leave any spaces (Carriage returns) between the lines

rojasja committed:
Update active-directory-b2c-create-custom-attributes-profile-edit-custom.md

barclayn committed:
staging vendor article for Tom

barclayn committed:
fixing broken links

barclayn committed:
fixing broken links

bwren committed:
oops

bwren committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

bwren committed:
fix

ccompy committed:
Merge pull request #2 from MicrosoftDocs/master

update my master

ccompy committed:
updated information around Web Jobs on an ILB ASE

ccompy committed:
Updated text for web jobs and functions

barclayn committed:
minor changes to staged doc

rojasja committed:
trying to fix how code is shown in the custom doc

rojasja committed:
Updating custom attribute custom file CR fix

TomArcherMsft committed:
Per ghogen, I assigned Visual Studio articles to kraig that were previously assigned to tarcher

rodsan committed:
addressed issues from last PR, added additional md files and images

rodsan committed:
made warning changes

rahulmsft committed:
Update sql-database-recovery-using-backups.md

Point in time restore on a geo-secondary is not currently supported. Point in time restores can be done only on the primary. We received a customer question regarding this, hence updating this in our documentation.

rahulmsft committed:
Update sql-database-business-continuity.md

Received specific customer query regarding this and we felt it was important to explicitly state this in our documentation. Point-in-time restore on geo-secondary is not currently supported.

rodsan committed:
Per comments from @Lisaco88 both blocking and nonblocking issues are fixed. Thanks!

carolinacmoravia committed:
CC6061: Duplicated paragraph

@mimig1, 
This proposed file change comes from this contribution https://github.com/MicrosoftDocs/azure-docs.it-it/pull/55/files. Could you review it?
Many thanks in advance.

Commit:
ljljljkljkljl

bwren committed:
faq and known issues

bwren committed:
faq

bwren committed:
arm

bwren committed:
update

ccompy committed:
clarified private app access

MicrosoftGuyJFlo committed:
MFA HA Doc

MicrosoftGuyJFlo committed:
MFA HA Doc

MicrosoftGuyJFlo committed:
Updates from previous PR

barclayn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into discover-identity-personal-data

barclayn committed:
making changes per Tom

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fresh5

barclayn committed:
making minor changes

carolinacmoravia committed:
M10602: Duplicated content 

@kgremban, 
Coudl review the proposed file change, please? It seems that reported content is duplicated,
Many thanks in advance.

barclayn committed:
making changes

barclayn committed:
replacing powershell image with code snippet

barclayn committed:
fixing minor mistakes

ccompy committed:
clarified pricing

rwike77 committed:
freshness updates

ccompy committed:
improved acrolinx stats

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

ccompy committed:
fixed a typo

kgremban committed:
Update active-directory-application-proxy-claims-aware-apps.md

cwatson-cat committed:
Added new article, screenshots, and updated the TOC.

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mmacy committed:
Merge branch 'master' of https://github.com/mmacy/azure-docs-pr

cwatson-cat committed:
Added appropiate bolding for UI text.

bwren committed:
updates

ErikjeMS committed:
light rewrite

ErikjeMS committed:
Update azure-stack-marketplace.md

rwike77 committed:
more updates

rwike77 committed:
tweeks

barclayn committed:
staging last two articles

ccompy committed:
format fix

barclayn committed:
trying to fix some errors

ccompy committed:
more typo fixes

barclayn committed:
fixing a broken link

ggailey777 committed:
Fixed typo

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fresh5

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

cwatson-cat committed:
Updated ms custom on three articles and fixed meta-data description on new topic.

cwatson-cat committed:
Fixed toc so resource manger is upper case.

ErikjeMS committed:
Update azure-stack-marketplace.md

cwatson-cat committed:
Trying to fix weird image alignment

TomArcherMsft committed:
Also assigning storage articles to KraigB

cwatson-cat committed:
Trying to fix screenshot alignment again.

TomArcherMsft committed:
Merge branch 'master' into move-vs-articles-to-kraigb

cwatson-cat committed:
For Fs sakes this better fix the image alignment.

ErikjeMS committed:
Update azure-stack-marketplace.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fresh5

Commit:
Merge pull request #20278 from kgremban/patch-1

Update active-directory-application-proxy-claims-aware-apps.md

Commit:
Merge pull request #20209 from carolinacmoravia/patch-360

M10602: Duplicated content

CarlRabeler committed:
Update sql-database-cloud-migrate.md

Commit:
updates to try and make the IP addresses you need clearer

itechedit committed:
Merge pull request #20283 from cwatson-cat/8-24-highcontrast-azureportalthemes

Added new Azure portal themes article, screenshots, and updated the TOC.

Commit:
freshness review

Lisaco88 committed:
Merge pull request #19829 from TomArcher/move-vs-articles-to-kraigb

move-vs-articles-to-kraigb

Commit:
Merge pull request #20385 from CarlRabeler/patch-325

Update sql-database-cloud-migrate.md

Commit:
Merge pull request #20386 from Blackmist/fresh5

updates for support

Commit:
Merge pull request #20359 from ggailey777/patch-6

Fixed typo

rachelappel committed:
merge

Lisaco88 committed:
Merge pull request #20381 from ErikjeMS/master

update article

Commit:
Merge pull request #20388 from msebolt/iot-hub-operations-monitoring

freshness review

rojasja committed:
Additional fixes to formatting

Commit:
Merge pull request #19869 from rahulmsft/patch-1

Update sql-database-recovery-using-backups.md

ErikjeMS committed:
new file

Commit:
Merge pull request #19871 from rahulmsft/patch-2

Update sql-database-business-continuity.md

rmca14 committed:
Merge pull request #20306 from barclayn/last-two-articles-deb

staging last two articles

rachelappel committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into webapp-linux-remove

Commit:
freshness review

rachelappel committed:
toc

ErikjeMS committed:
Update azure-stack-offer-services-overview.md

davidmu1 committed:
incorp feedback

rojasja committed:
Fixed ADFS picture formatting

Added two spaces in previous line to terminate

rwike77 committed:
formatting updates

ccompy committed:
improved style

Commit:
Merge pull request #20390 from msebolt/iot-hub-devguide-direct-methods

freshness review

ErikjeMS committed:
Update azure-stack-offer-services-overview.md

Commit:
Merge pull request #19094 from carolinacmoravia/patch-315

M8230: Possible incorrect terminology: **Device settings > Hardware h…

davidmu1 committed:
Merge branch 'master' into java

ErikjeMS committed:
Update azure-stack-offer-services-overview.md

mimig1 committed:
Cosmos DB: Updated next steps links.

Commit:
Merge pull request #19370 from carolinacmoravia/patch-335

M8269: Incorrect image structure

barclayn committed:
addressing issues identified by the reviewers

Commit:
Merge pull request #19398 from carolinacmoravia/patch-338

M8339: Repeated word: "policy"

itechedit committed:
Merge pull request #18282 from rachelappel/webapp-linux-remove

Remove app-service-linux-how-to-create-web-app doc, update toc

CarlRabeler committed:
Update sql-database-features.md

CarlRabeler committed:
Update sql-database-features.md

Commit:
Merge pull request #20392 from MicrosoftDocs/mimig1-patch-2

Cosmos DB: Updated next steps links.

Commit:
Merge pull request #20398 from CarlRabeler/patch-326

Update sql-database-features.md

Commit:
freshness review

Commit:
Merge pull request #19408 from carolinacmoravia/patch-340

M8342: Possible incorrect string: "Configure App Service web apps wit…

Commit:
fix query and remove incorrect screenshot

CarlRabeler committed:
Update sql-database-technical-overview.md

Commit:
Merge pull request #19410 from carolinacmoravia/patch-341

M8355: Incorrect bold type structure

tfitzmac committed:
changed template links

rwike77 committed:
fixed numbering

Commit:
Merge pull request #19415 from carolinacmoravia/patch-343

M8356: Incorrect tag that block text content

Commit:
Merge pull request #20400 from msebolt/iot-hub-device-sdk-c-intro

freshness review

Commit:
Merge pull request #20402 from tfitzmac/0825changetemplate

changed template links

Commit:
Merge pull request #20063 from carolinacmoravia/patch-357

CC6061: Duplicated paragraph

Commit:
fix ASA forum link

rwike77 committed:
formatting

bwren committed:
fixes

ErikjeMS committed:
new article

Ja-Dunn committed:
Merge pull request #19694 from barclayn/discover-identity-personal-data

staging vendor article for Tom

CarlRabeler committed:
Update sql-database-technical-overview.md

ErikjeMS committed:
Update azure-stack-offer-services-overview.md

Commit:
freshness review

Ja-Dunn committed:
Merge pull request #19607 from rojasja/master

Updating docs and new IDP scenarios -2nd

Ja-Dunn committed:
Merge pull request #20295 from rwike77/freshness

Freshness updates

CarlRabeler committed:
Update toc.yml

ccompy committed:
more acronlinx related improvements

Ja-Dunn committed:
Merge pull request #20100 from MicrosoftGuyJFlo/MFAHADocPublish

MFA HA Doc

a-tan committed:
Typo update

Azure Data Lake Storage -> Azure Storage

ShawnJackson committed:
Update app-service-mobile-xamarin-forms-get-started.md

ErikjeMS committed:
Update azure-stack-offer-services-overview.md

rmca14 committed:
Merge pull request #17471 from davidmu1/java

new article

Commit:
Merge pull request #20399 from CarlRabeler/patch-327

Update sql-database-technical-overview.md

ShawnJackson committed:
Update app-service-mobile-configure-new-backend.md

Ja-Dunn committed:
Merge pull request #20401 from minhe-msft/master

fix query and remove incorrect screenshot

Ja-Dunn committed:
Merge pull request #20407 from CarlRabeler/patch-328

Update toc.yml

Commit:
Merge pull request #20409 from a-tan/patch-1

Typo update

Ja-Dunn committed:
Merge pull request #20079 from bwren/kusto-upgrade

Kusto faq and known issues

andredm7 committed:
Changing ownership

Commit:
Merge pull request #20405 from msebolt/iot-hub-python-getstarted

freshness review

Ja-Dunn committed:
Merge pull request #20411 from andredm7/andredm0825171

Changing ownership

Ja-Dunn committed:
Merge pull request #20404 from ErikjeMS/master

adding offer overview article

Commit:
freshness review

rmca14 committed:
Merge pull request #19833 from rodsan/master

Addressed issues from last PR, added additional files and images + TOC update

ShawnJackson committed:
Merge pull request #19503 from itechedit/app-service-mobile-xamarin-forms-get-started

edit pass: app-service-mobile-xamarin-forms-get-started

mmacy committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge pull request #20415 from msebolt/iot-hub-monitoring-notifications-with-azure-logic-apps

freshness review

Commit:
freshness review

ShawnJackson committed:
Update active-directory-saas-helpscout-tutorial.md

mmacy committed:
[aci] restore Templates node under Resources

Juliako committed:
updated the manager contact info

ShawnJackson committed:
Merge pull request #19532 from ktoliver/active-directory-saas-helpscout-tutorial

edit pass: active-directory-saas-helpscout-tutorial.md

MicrosoftGuyJFlo committed:
Update to MFA TOC

Juliako committed:
Update TOC.md

Commit:
Merge pull request #20416 from msebolt/iot-hub-scaling

freshness review

tfitzmac committed:
added stream analytics limitation

Commit:
freshness review

Ja-Dunn committed:
Merge pull request #20418 from Juliako/RESTupdates

updated the manager contact info

Ja-Dunn committed:
Merge pull request #20417 from mmacy/aci-resources-add

[aci] restore Templates node under Resources

Lisaco88 committed:
Merge pull request #20249 from ccompy/update-ilb-ase-doc

Updated several docs for ASEv2

Ja-Dunn committed:
Merge pull request #20421 from Juliako/patch-13

Update TOC.md

Ja-Dunn committed:
Merge pull request #20420 from MicrosoftGuyJFlo/MFATOCUpdate

Update to MFA TOC

Commit:
Merge pull request #20422 from tfitzmac/0825move

added stream analytics limitation

ShawnJackson committed:
Update active-directory-saas-sapboc-tutorial.md

Commit:
Merge pull request #20423 from msebolt/iot-hub-node-node-direct-methods

freshness review

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ShawnJackson committed:
Update active-directory-saas-sapboc-tutorial.md

v-alje committed:
Merge pull request #20425 from MicrosoftDocs/FromPublicRepo

From public repo

ShawnJackson committed:
Merge pull request #19635 from ktoliver/active-directory-saas-sapboc-tutorial

edit pass: active-directory-saas-sapboc-tutorial.md
